### DISCUSSION TITLE
Trying out Reactivated without having to install Nix

### DISCUSSION THREAD

**Discussion by thibaudcolas** [silviogutierrez/reactivated#156] [Q&A] [answered]
👋  I’m really interested by the prospect of using React / JSX as a replacement for Django’s template layer, but not so keen on Nix. I read the [Why Nix](https://www.reactivated.io/documentation/why-nix/) page in the documentation, it all makes sense to me, but I’m happy with my current approach to dependency management.

Are you intending to offer a way to use Reactivated without Nix? Is there some way I could try this out with a bit more legwork for the project setup? I see there is a `create-` package on npm which looks to be nix-only from the source, and then I assume I will be able to use the `reactivated` packages on PyPI and npm without Nix? Any pointers would be appreciated!

**Comment by silviogutierrez** [comment, marked as answer]
Certainly, you can use Reactivated without Nix. It's just going to be more trial and error. It's documented here: https://www.reactivated.io/documentation/existing-projects/

I recommend downloading Django 4 using whatever dependency management tool you like. Then after running `django-admin startproject`, give the above instructions a try. Theoretically, it should *just work*. But I'm happy to help you diagnose any issues. Then I can also fix the documentation.

Hope that helps.

**Reply by thibaudcolas** [reply]
Perfect! I didn’t think of checking that page, thank you.

**Reply by silviogutierrez** [reply]
There's a pretty decent Docker experience now as well: https://www.reactivated.io/documentation/getting-started/#with-docker
